modules/redisearch.ts

```diff
@@ -39,51 +39,56 @@ export class Redisearch extends Module {
-            if(parameters.noOffsets !== undefined)
-                args.push('NOOFFSETS');
+                args = args.concat(['MAXTEXTFIELDS', `${parameters.maxTextFields}`])
             if(parameters.temporary !== undefined)
-                args.push('TEMPORARY');
-            if(parameters.nohl !== undefined)
-                args.push('NOHL');
-            if(parameters.noFields !== undefined)
-                args.push('NOFIELDS');
-            if(parameters.noFreqs !== undefined)
-                args.push('NOFREQS');
-            if(parameters.stopwords !== undefined)
-                args = args.concat(['STOPWORDS', parameters.stopwords.num.toString(), parameters.stopwords.stopword]);
-            if(parameters.skipInitialScan !== undefined)
-                args.push('SKIPINITIALSCAN');
+                args = args.concat(['TEMPORARY', `${parameters.temporary}`])
+            if(parameters.noOffsets === true)
+                args.push('NOOFFSETS')
+            if(parameters.nohl === true)
+                args.push('NOHL')
+            if(parameters.noFields === true)
+                args.push('NOFIELDS')
+            if(parameters.noFreqs === true)
+                args.push('NOFREQS')
+            if(parameters.stopwords !== undefined) {
+                args.push('STOPWORDS')
+                args.push(`${parameters.stopwords.length}`)
+                args = args.concat(parameters.stopwords)
+            }
+            if(parameters.skipInitialScan === true)
+                args.push('SKIPINITIALSCAN')
         }
         args.push('SCHEMA');
         for(const field of schemaFields) {
             args.push(field.name)
-            if(field.as)
+            if (field.as !== undefined)
                 args = args.concat(['AS', field.as])
             args.push(field.type);
-            if(field.nostem !== undefined) args.push('NOSTEM');
-            if(field.weight !== undefined) args = args.concat(['WEIGHT', field.weight.toString()]);
-            if(field.phonetic !== undefined) args = args.concat(['PHONETIC', field.phonetic]);
-            if(field.seperator !== undefined) args = args.concat(['SEPERATOR', field.seperator]);
-            if(field.sortable !== undefined) args.push('SORTABLE');
-            if(field.noindex !== undefined) args.push('NOINDEX');
+            if (field.nostem === true) args.push('NOSTEM');
+            if (field.weight !== undefined) args = args.concat(['WEIGHT', `${field.weight}`])
+            if (field.phonetic !== undefined) args = args.concat(['PHONETIC', field.phonetic])
+            if (field.seperator !== undefined) args = args.concat(['SEPERATOR', field.seperator])
+            if (field.sortable === true) args.push('SORTABLE')
+            if (field.noindex === true) args.push('NOINDEX')
+            if (field.unf === true) args.push('UNF')
+            if (field.caseSensitive === true) args.push('CASESENSITIVE')
```

Comment:
Can you please remove the space after the `if` statement, I prefer it that and I like consistency.

Reply:
With your new eslint file, this should be taken care of, automatically 😉